packages/suite/src/views/wallet/coinmarket/savings/setup/components/Summary/index.tsx

```diff
@@ -27,7 +27,7 @@ const Right = styled.div`
     flex-direction: column;
 `;
 
-const Fiat = styled.div`
+const Fiat = styled(HiddenPlaceholder)`
```

Comment:
how about just using the `FiatValue` component here instead?

Reply:
Would like to but `FiatValue` uses default `FiatAmountFormatter` without posibillity to override `maximumFractionDigits` and `minimumFractionDigits`. Suggestions?

Reply:
Isn't it used [here](https://github.com/trezor/trezor-suite/pull/7017/files#diff-d6e657270272496547a315b71f9183f9c261f13cb554c4557d8192505c4d325eR59) anyways? 🤔

Reply:
Component `FiatValue` is not used in mentioned [changes](https://github.com/trezor/trezor-suite/pull/7017/files#diff-d6e657270272496547a315b71f9183f9c261f13cb554c4557d8192505c4d325eR59).

Reply:
Or what do you mean by "it"?

Reply:
I mean the `FiatAmountFormatter`

Reply:
The fix to resolve this issue is to use such component which hides/anonymizes the amount. It is OK to use `FiatValue` to solve this issue, but since there is a need to use custom `FiatAmountFormatter` the added value of using `FiatValue` is gone because `FiatValue`does not render `HiddenPlaceholder` when there is need to render children components.

Yes, there is solution like this:

```
                                <FiatValue
                                    amount={savingsTrade.fiatStringAmount || '0'}
                                    symbol={symbol}
                                    fiatCurrency={savingsTrade.fiatCurrency?.toLowerCase()}
                                >
                                    {() => (
                                        <FiatPaymentWrapper>
                                            <FiatAmountFormatter
                                                value={savingsTrade.fiatStringAmount || '0'}
                                                currency={savingsTrade.fiatCurrency}
                                                maximumFractionDigits={2}
                                                minimumFractionDigits={0}
                                            />
                                        </FiatPaymentWrapper>
                                    )}
                                </FiatValue>
```
Where `FiatPaymentWrapper` is actually styled `HiddenPlaceholder`. Do we really want this? I really do not see the solution while using `FiatValue` and `FiatAmountFormatter` and `HiddenPlaceholder` together. Please advise. We can also arrange call. (Looks like my React skills are poorer as I did not implement a thing in React in weeks [😅](https://emojipedia.org/grinning-face-with-sweat/))

Reply:
I realized I had an incorrect link in my previous comment. I mean that the plain `FiatAmountFormatter` is used in this [Summary component](https://github.com/trezor/trezor-suite/blob/fc002ae043dd16eae7988e7e3302728b54842182/packages/suite/src/views/wallet/coinmarket/savings/setup/components/Summary/index.tsx#L59). I am not sure where a _custom formatter_ is used, maybe I am missing something? And I think that `FiatValue` does [render](https://github.com/trezor/trezor-suite/blob/fc002ae043dd16eae7988e7e3302728b54842182/packages/suite/src/components/suite/FiatValue.tsx#L85) the `HiddenPlaceholder` unless it's explicitly disabled.

Reply:
All clear after our call, thanks :)